### ISSUE TITLE
EXDEV: cross-device link not permitted (on Linux)

### ISSUE THREAD

**Issue by makkus** [runtools/run#1]
Hiya, just gave run a go, ran (no pun intended) into an issue on Linux:

```
❯ run @registry
{ Error: EXDEV: cross-device link not permitted, rename '/tmp/8d44d3f1dc023f4de9ac635acc6eeb55' -> '/home/markus/.run/resdir-registry/resource-cache/resdir/registry-client/versions/0.1.27'
    at Error (native)
    at fs.renameSync (fs.js:811:18)
    at /Users/mvila/Projects/resdir/public/packages/resource-fetcher/src/index.js:222:7
    at next (native)
    at step (/usr/local/lib/node_modules/run-cli/node_modules/@resdir/resource-fetcher/dist/node/cjs/index.js:36:221)
    at _next (/usr/local/lib/node_modules/run-cli/node_modules/@resdir/resource-fetcher/dist/node/cjs/index.js:36:409)
    at /usr/local/lib/node_modules/run-cli/node_modules/@resdir/resource-fetcher/dist/node/cjs/index.js:36:477
    at /usr/local/lib/node_modules/run-cli/node_modules/@resdir/resource-fetcher/dist/node/cjs/index.js:36:97
    at ResourceFetcher._saveCachedResourceVersion (/Users/mvila/Projects/resdir/public/packages/resource-fetcher/src/index.js:213:89)
    at /Users/mvila/Projects/resdir/public/packages/resource-fetcher/src/index.js:90:23
  errno: -18,
  code: 'EXDEV',
  syscall: 'rename',
  path: '/tmp/8d44d3f1dc023f4de9ac635acc6eeb55',
  dest: '/home/markus/.run/resdir-registry/resource-cache/resdir/registry-client/versions/0.1.27',
  contextStack: 
   [ Resource {
       '$defaultAutoBoxing': false,
       '$defaultAutoUnboxing': false,
       '_$bases': [],
       '_$children': [],
       '_$currentDirectory': '/home/markus',
       '_$isInitializing': false,
       '_$listeners': {},
       '_$hasBeenInitialized': true } ] }
```
I don't really know much node, so maybe this is a known issue and my partition setup is at fault. But figured it can't hurt to log it...

**Comment by mvila** [comment]
Thanks, @makkus for posting issue #1! 🎉 

That was a bug; I was trying to move a directory from the temporary directory to the user directory without thinking it could be on different volumes.

It should be fixed in `run-cli@0.11.10`, to update just do:

`npm install run-cli -g`

Can you confirm that your problem is solved, please?

**Comment by makkus** [comment]
Yup, all works now. Cheers!
